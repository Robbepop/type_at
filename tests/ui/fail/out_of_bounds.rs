use type_at::TypeAt;

fn main() {
    let _: _  = <(i8,) as TypeAt<1>>::Type::default();
}
