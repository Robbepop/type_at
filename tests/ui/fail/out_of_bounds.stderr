error[E0277]: the trait bound `(i8,): TypeAt<1_usize>` is not satisfied
 --> $DIR/out_of_bounds.rs:4:17
  |
4 |     let _: _  = <(i8,) as TypeAt<1>>::Type::default();
  |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `TypeAt<1_usize>` is not implemented for `(i8,)`
  |
  = help: the following implementations were found:
            <(A,) as TypeAt<0_usize>>
